components/optimizely-subdomain.jsx

```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var optimizelyActive = process.env.OPTIMIZELY_ACTIVE || 'yes';
+var cookieDomain = process.env.FULL_SUBDOMAIN_FOR_COOKIE || 'donate.mozilla.org';
+
+var OptimizelySubdomain = React.createClass({
+  cookieScript: function() { return {__html: "window['optimizely'] = window['optimizely'] || []; window['optimizely'].push(['setCookieDomain', '" + cookieDomain + "']);"}; },
+  render: function() {
+    return optimizelyActive ? <script dangerouslySetInnerHTML={this.cookieScript()} /> : false;
```

Comment:
@Pomax I tried but it seems like special characters were escaped -- source code shows 
```html
<script>
window[&#x27;optimizely&#x27;] = window[&#x27;optimizely&#x27;] || []; 
window[&#x27;optimizely&#x27;].push([&#x27;setCookieDomain&#x27;, &#x27;teach.mozilla.org&#x27;]);
</script>
```